com.github.jeremypw.gnonograms.json

```diff
@@ -0,0 +1,45 @@
+{
+    "app-id": "com.github.jeremypw.gnonograms",
+    "base": "io.elementary.BaseApp",
+    "base-version": "juno-19.08",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "com.github.jeremypw.gnonograms",
+
+    "finish-args": [
+      "--share=ipc",
+      "--socket=fallback-x11",
+      "--socket=wayland",
+      "--filesystem=home",
+      "--device=dri"
+    ],
+
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+
+    "modules": [
+        {
+            "name": "gnonograms",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/jeremypw/gnonograms.git",
```

Comment:
elementary apps tend to enforce their stylesheet/icons otherwise they look half broken with other themes. Is it the case with your application?

Reply:
Good question. It supplies some custom icons and adds them to the default Gtk theme. It does not enforce the elementary theme. I think it should adapt to other themes OK but I will try it out. I have converted to draft for now while I verify.